internal/exec/resolvable/packer.go

```diff
@@ -130,8 +130,11 @@ func (b *execBuilder) makeStructPacker(values common.InputValueList, typ reflect
 	var fields []*structPackerField
 	for _, v := range values {
 		fe := &structPackerField{field: v}
+		fx := func(n string) bool {
+			return strings.EqualFold(n, v.Name.Name) || strings.EqualFold(camelize(n), camelize(v.Name.Name))
```

Comment:
@shurcooL  Yup, I think that's not necessary, I was thinking to preserve original comparison. Removed in https://github.com/neelance/graphql-go/pull/77/commits/416031fa414f09c20d97281d9da93b90ab1e3e52.